salt/iiif/config/opt-loris-loris2.conf

```diff
@@ -28,6 +31,24 @@ enable_caching = True
 redirect_canonical_image_request = False
 redirect_id_slash_to_info = True
 
+# max_size_above_full restricts interpolation of images on the server.
+# Default value 200 means that a user cannot request image sizes greater than
+# 200% of original image size (width or height).
+# Set this value to 100 to disallow interpolation. Set to 0 to remove
+# size restriction.
+
+# lsh@2021-06-28: enabled by default but disabled until properly evaluated
```

Comment:
if this is enabled by default then by *disabling* it I'm actually modifying settings ... revert